options/base_options.py

```diff
@@ -661,6 +663,12 @@ def initialize(self, parser):
             help="scaling and cropping of images at load time",
         )
 
+        parser.add_argument(
```

Comment:
This should go with the remaining sam options, in palette ?

Reply:
I don't think it should since sam model is initialized in base_diffusion_model.py